.travis.yml

```diff
@@ -11,14 +11,18 @@ mono: none
 dotnet: 2.2.300
 
 before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt install cloc; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install cloc; fi
   - cd neo.UnitTests
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then ulimit -n 2048; fi
 
 script:
  - dotnet restore
  - find * -name *.csproj | xargs -I % dotnet add % package coverlet.msbuild
  - dotnet test /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
-
+ - echo "Reporting number of lines with cloc"
+ - cloc ..
```

Comment:
Where is this info going?

Reply:
only travis for now, a bot could report it, but I think that on travis is enough

Reply:
We need it?